src/components/CommentsList.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from "react"
+import Loading from "./Loading";
+import { getCommentsByArticleId } from "../../api";
+import CommentCard from "./CommentCard";
+import { useParams } from "react-router-dom";
+
+function CommentsList(props) {
+
+    const { setIsLoading, isLoading } = props;
+    const [currentComments, setCurrentComments] = useState([])
+    const { article_id } = useParams();
+
+    useEffect(() => {
+        setIsLoading(true)
+        getCommentsByArticleId(article_id).then((comments) => {
+            setCurrentComments(comments);
+            setIsLoading(false)
+        });
+    }, []);
+
+    if (isLoading) {
+        return <Loading></Loading>;
```

Comment:
this should be self-closing if it does not require children to be passed through